iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/partition/PartitionMetrics.java

```diff
@@ -274,23 +326,11 @@ public static void bindDatabasePartitionMetrics(
         database,
         Tag.TYPE.toString(),
         TConsensusGroupType.DataRegion.toString());
+    bindDatabaseReplicationFactorMetrics(
```

Comment:
If put this line in it, we should revise the function bindDatabasePartitionMetricsWhenUpdate name?